packages/google_maps_flutter/google_maps_flutter_android/android/src/main/java/io/flutter/plugins/googlemaps/Convert.java

```diff
@@ -703,6 +703,19 @@ static String interpretPolygonOptions(Messages.PlatformPolygon polygon, PolygonO
     return polygon.getPolygonId();
   }
 
+  static int jointTypeFromPigeon(Messages.PlatformJointType jointType) {
+    switch (jointType) {
+      case MITERED:
+        return 0;
+      case BEVEL:
+        return 1;
+      case ROUND:
+        return 2;
+    }
+    // These integer values come from a different package. This fallback ensures some value is returned in case a new enum value is added.
```

Comment:
This is not correct at this layer; `PlatformJointType` is from this package. If there's a case defined in `messages.dart` but not handled above that's a programming error, not a plausible runtime error.

Reply:
I see. I will remove the comment, but I believe we still need the return statement for Java compilation.